object-construction-checker/tests/mustcall/MustCallChoiceImpl.java

```diff
@@ -0,0 +1,24 @@
+// A simple test that the extra obligations that MustCallChoice imposes are
+// respected.
+
+// @skip-test until the checks are implemented
```

Comment:
@Nargeshdb can you comment here on the unexpected error you got for this file, so @kelloggm can take a look?